modules/huddledmassesBidAdapter.js

```diff
@@ -0,0 +1,156 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'huddledmasses';
+const URL = '//huddledmassessupply.com/?c=o&m=multi';
+const URL_SYNC = '//huddledmassessupply.com/?c=o&m=cookie';
+const SIZE_INDEX = {
+  '468x60': 1,
+  '728x90': 2,
+  '300x600': 10,
+  '300x250': 15,
+  '300x100': 19,
+  '320x50': 43,
+  '300x50': 44,
+  '300x300': 48,
+  '300x1050': 54,
+  '970x90': 55,
+  '970x250': 57,
+  '1000x90': 58,
+  '320x80': 59,
+  '640x480': 65,
+  '320x480': 67,
+  '320x320': 72,
+  '320x160': 73,
+  '480x300': 83,
+  '970x310': 94,
+  '970x210': 96,
+  '480x320': 101,
+  '768x1024': 102,
+  '1000x300': 113,
+  '320x100': 117,
+  '800x250': 118,
+  '200x600': 119
+};
+const INDEX_SIZE = {
```

Comment:
You don't need to duplicate this data. You can do a reverse lookup by iterating the object properties.